client/cli/src/params/import_params.rs

```diff
@@ -132,25 +134,25 @@ impl ImportParams {
 /// Execution strategies parameters.
 #[derive(Debug, StructOpt, Clone)]
 pub struct ExecutionStrategiesParams {
-	/// The means of execution used when calling into the runtime while syncing blocks.
+	/// The means of execution used when calling into the runtime while import locally authored blocks.
 	#[structopt(
-		long = "execution-syncing",
+		long = "execution-own-block-import",
```

Comment:
```suggestion
		long,
```
The name is automatically generated based on the field already